src/config.json

```diff
@@ -241,7 +241,9 @@
         "name": "viya",
         "serverType": "SASVIYA",
         "serverUrl": "https://sas.analytium.co.uk",
-        "allowInsecureRequests": false,
+        "httpsAgentOptions": {
+          "allowInsecureRequests": false
+        },
```

Comment:
CLI is accepting all attributes which can be passed to httpAgent i.e. `ca`, `cert`, `key`, `rejectUnauthorized` etc.

In addition to these, schema also accepts `caPath`, `certPath`, `keyPath`:
- to extract contents and pass to `ca`, `cert`, `key`

Hence `allowInsecureRequests`:
- based on it's value, cli setting `rejectUnauthorized`